.jenkins/pytorch/test.sh

```diff
@@ -274,6 +274,14 @@ test_libtorch() {
     else
       "$TORCH_BIN_DIR"/test_jit  --gtest_filter='-*CUDA' --gtest_output=xml:$TEST_REPORTS_DIR/test_jit.xml
     fi
+
+    # Run Lazy Tensor cpp tests
```

Comment:
mostly just wondering - were the cpp tests no running in CI before this? And any reason to only enable them for cuda builds on CI? (once the XLA plugin is ready, I'm guessing we want it to run on XLA too?)

Reply:
correct, it was an oversight, but not a big deal since we mostly looked at the lazy_tensor_staging CI, and also this test was running in sandcastle.  But it did confuse me for a while when I was trying to find the output of the expanded test suite in my later diff!

also note, this change is supposed to be landed first in https://github.com/pytorch/pytorch/pull/74449

I'm not sure why the rebase didn't work as intended, but this diff should only be adding the per-operator-headers stuff, and i'll land the other one first and make sure this one is clean.